app/javascript/react/screens/App/Overview/screens/MappingWizard/__tests__/__snapshots__/index.test.js.snap

```diff
@@ -23,7 +23,7 @@ Object {
       },
     },
     "syncErrors": Object {
-      "name": "Required",
+      "name": "Required. Maximum is 24 characters.",
```

Comment:
could be a nit and covered in a subsequent PR.  @vconzola I was just poking around and i wonder if we should state "Maximum length is 24 characters." what do ya think?

Reply:
I think either one works, "the maximum" for a string is obviously the max length, if we needed to save space. But we have plenty of horizontal space here so I agree that maybe "Maximum length is..." is better here.